src/components/UserSelector.tsx

```diff
@@ -1,6 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { User } from '../types/User';
+
+type Props = {
+  users: User[] | null;
+  currUser: User | null;
+  selectUser: (user: User) => void;
+};
+
+export const UserSelector: React.FC<Props> = ({
+  users, currUser, selectUser,
+}) => {
+  const [isListShown, setListShown] = useState(false);
+
+  const toggleList = () => {
+    setListShown(!isListShown);
+  };
+
+  const hideList = () => {
```

Comment:
Consider removing the 'hideList' function and controlling the list visibility using only the 'toggleList' function.